be/src/common/status.cpp

```diff
@@ -4,10 +4,35 @@
 
 #include "common/status.h"
 
-#include "gutil/strings/fastmem.h" // for memcpy_inlined
+#include <boost/stacktrace.hpp>
+#include <glog/logging.h>
 
+#include "gutil/strings/fastmem.h" // for memcpy_inlined
 namespace doris {
 
+constexpr int MAX_ERROR_NUM = 65536;
+struct ErrorCodeState {
+    int16_t error_code = 0;
+    bool stacktrace = true;
+    std::string description;
+    size_t count = 0;   // Used for count the number of error happens
+    std::mutex mutex;   // lock guard for count state
```

Comment:
why not use atomic？

Reply:
This filed is not used yet. Just add a filed and used in the future and I could remove it.